docs/guides/persistence.mdx

```diff
@@ -73,6 +73,68 @@ const atomWithAsyncStorage = (key, initialValue) => {
 }
 ```
 
+### Usage in Async actions
+
+When using an asynchronous storage, the reading of the created atoms are asynchronous too. That comes with some limitations. Before `useAtom` has been called at least once, the value is not set.
+This can cause an issue, for example, if you want to retrieve the value of your stored atom in an async action (write-only atom).
+
+Let's use the [atomWithStorage function in the utils bundle](../api/utils.mdx#atom-with-storage) for the examples, so that we have more options available.
+
+```typescript
+const storage = createJSONStorage(() => AsyncStorage)
+const userId = atomWithStorage('user-id-key', null, storage)
+const userInfo = atom({})
+const fetchUserInfo = atom(null, async (_get, set, payload: any) => {
+    const id = _get(userId) // This throws an error if "userId" is not loaded yet
```

Comment:
Please just use `get`. `_get` is a convention for "unused" value.

Reply:
@dai-shi Oh, right, makes sense! I was wondering. Is this mentioned somewhere?

Reply:
For example, TypeScript `noUnusedParameters` option will skip `_foo`.
https://stackoverflow.com/questions/41685054/clarification-on-typescripts-nounusedparameters-compiler-option